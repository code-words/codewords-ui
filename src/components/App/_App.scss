.backdrop {
  content: "";
  background: $overlay-filter;
  position: absolute;
  top: 0px;
  height: 100%;
  width: 100%;
  display: flex;
  justify-content: center;
  align-items: center;
}

.btn-default {
  border-radius: 20px;
  display: flex;
  align-items: center;
  justify-content: center;
  font-size: 16px;
  color: white;
  cursor: pointer;
  background: $button-default;
  text-decoration: none;

  &:hover {
    background: $button-hover;
  }

  &:disabled {
    color: gray;
    cursor: default;
    font-size: 1em;
    font-weight: 400;
    background: $button-default;
  }
}