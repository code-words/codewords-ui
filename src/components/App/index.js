import React from 'react';
import { Route } from 'react-router-dom';
import Header from '../Header';
import StartScreen from '../../StartScreen';
import RuleList from '../RuleList';
import NewGame from '../NewGame';
import JoinGame from '../JoinGame';
import Lobby from '../Lobby';
import Main from '../Main';
import ConversationsList from '../ConversationsList';

function App() {
  return (
    <div className="App">
      <Header />
      <Route exact path='/startScreen' component={StartScreen} />
      <Route exact path='/ruleList' component={RuleList}/>
      <Route exact path='/newGame' component={NewGame} />
      <Route exact path='/joinGame' component={JoinGame} />
      <Route exact path='/lobby' component={Lobby} />
      <Route path='/' component={Main} />
      <ConversationsList />
    </div>
  );
}

export default App;
