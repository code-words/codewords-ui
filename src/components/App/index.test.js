import React from 'react';
import App from './index';
import { shallow } from 'enzyme';

describe('App', () => {
  let wrapper;
  let instance;

  beforeEach(() => {
    wrapper = shallow(<App />);
    instance = wrapper.instance();

  })

  it('should match component snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

})