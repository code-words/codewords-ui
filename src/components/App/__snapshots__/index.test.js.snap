// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match component snapshot 1`] = `ShallowWrapper {}`;
